.github/workflows/build-plugin-archive.yml

```diff
@@ -202,16 +215,27 @@ jobs:
           composer global require inpsyde/wp-translation-downloader
           composer wp-translation-downloader:download
 
+      - name: Run PHP Scoper
+        if: steps.composer-tools.outputs.php-scoper == '0'
+        run: |
+          php-scoper add-prefix --force --output-dir build
+          composer --working-dir=build dump-autoload -o
+          sed -i "s/'__composer_autoload_files'/\'__composer_autoload_files_${{ github.sha }}'/g" "build/vendor/composer/autoload_real.php"
+      - name: Move unchanged code to build directory
+        if: steps.composer-tools.outputs.php-scoper != '0'
+        run: mv * build 2>/dev/null
```

Comment:
Looks like it does not work, probably because the `build` dir does not exist.

Fixed in another branch https://github.com/inpsyde/reusable-workflows/commit/3ba25630c7e7edc030f74a625618d37e4bb6280b
Not sure if there is a better way and if `shopt -s extglob` can cause issues.